tests/experimental/nn/test_sync_batchnorm.py

```diff
@@ -92,6 +97,34 @@ def parity3d_bn():
     check_parity(torch_bn, fs_bn, x)
 
 
+@pg_test()
+def parity3d_checkpoint_syncbn():
+    rank = dist.get_rank()
+    torch.cuda.set_device(rank)
+    torch.manual_seed(rank)
+
+    x = torch.randn(4, 3, 4, 4, 4).cuda() * rank
+    torch_bn = torch.nn.SyncBatchNorm(3).cuda()
+    fs_bn = SyncBatchNorm(3).cuda()
+    fs_bn = checkpoint_wrapper(fs_bn, maintain_forward_counter=True)
+    check_parity_ddp(torch_bn, fs_bn, x)
+
+
+@pg_test()
+def parity3d_checkpoint_syncbn_twice():
```

Comment:
how does this test 2 forward passes?

Reply:
This tests uses an nn.Sequential to call the same BatchNorm instance twice.

Reply:
ah got it! it's the same instance. For some reason I was thinking of two explicit forward calls and checking stats in between.